utils/auth.ts

```diff
@@ -10,8 +10,6 @@ export const AUTH_WHITELIST = ['/login', '/access-denied'];
 
 const GSSO_TOKEN_NAME = process.env.GSSO_TOKEN_NAME;
 
-export const SESSION_EXPIRY = 14400;
```

Comment:
I'm not sure this is right @mapleleafjack we should still be checking the expiry.

Reply:
Re-added!